clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/restapiv2/StateRestApiTest.java

```diff
@@ -42,8 +42,8 @@ protected void setUp(boolean dontInitializeNode2) throws Exception {
         {
             Set<ConfiguredNode> nodes = FleetControllerTest.toNodes(0, 1, 2, 3);
             ContentCluster cluster = new ContentCluster(
-                    "books", nodes, distribution, 6 /* minStorageNodesUp*/, 0.9 /* minRatioOfStorageNodesUp */,
-                    true /* determineBucketsFromBucketSpaceMetric */);
+                    "books", nodes, distribution, 6 /* minStorageNodesUp*/, 0.9 /* minRatioOfStorageNodesUp */
+                    /* determineBucketsFromBucketSpaceMetric */);
```

Comment:
Remove comment for the removed argument as well

Reply:
Fixed